core/js/src/test/scala/com/lynbrookrobotics/potassium/JSClockTest.scala

```diff
@@ -42,4 +42,17 @@ class JSClockTest extends AsyncFunSuite {
 
     future.map(t => assert(t <= 250))
   }
+
+  test("Single execution can be canceled") {
+    val testDonePromise = Promise[Unit]()
+    val future = testDonePromise.future
+    var executed = false
+
+    JSClock.singleExecution(Milliseconds(500)) {
+      throw new IllegalStateException("This code should not have run")
+    }()
+
+    testDonePromise.success(Unit)
```

Comment:
Instead of `Unit`, can you put `()`?

Reply:
Also, doing this makes your test immediately complete, which wouldn't catch bugs if your cancellation code was broken.